MApp.css

```diff
@@ -0,0 +1,42 @@
+body{
```

Comment:
Few comments about CSS here: 
1. You could include display flex or grid to body of html or a div that wraps the whole app. HTML elements are not organized due to lack of flexbox or grid. 
2. Many of style guideline provided in the assessment rubric is not followed. I'm not sure whether it was lack of time or lack of css skills but please keep practicing css since it's a must have skill for a developer. Reach out to me if you need any help. 
3. background colors are on point though.